src/turnkeyml/llm/docs/llamacpp.md

```diff
@@ -0,0 +1,38 @@
+# LLAMA.CPP
+
+Run transformer models using a Llama.cpp binary and checkpoint. This model can then be used with chatting or benchmarks such as MMLU.
+
+## Prerequisites
+
+This flow has been verified with a generic Llama.cpp model.
+
+### Set up Environment (Assumes TurnkeyML is already installed)
```

Comment:
Please mention that these instructions are only for Linux/ WSL or add instructions for Windows with cmake

Reply:
Done